### ISSUE TITLE
Look into alternate ML models

### ISSUE THREAD

**Issue by SophieMBerger** [ubclaunchpad/physiospace-react#24]

**Comment by jackyzha0** [comment]
Ok after doing a bit more research, there's two main parts we need to implement
1: Stacked Hourglass Network for 2D Pose Prediction (arxiv.org/abs/1603.06937)
Key takeaways can be found here: https://github.com/aleju/papers/blob/master/neural-nets/Stacked_Hourglass_Networks_for_Human_Pose_Estimation.md

Train on MPII: http://human-pose.mpi-inf.mpg.de/
Batch norm, single hourglass, RMSProp

2: Simple Linear NN with Batch Norm and Dropout following the work here (https://github.com/una-dinosauria/3d-pose-baseline)

**Comment by SophieMBerger** [comment]
@jackyzha0 Thanks so much for digging into this! It sounds pretty doable to me, what do you think?

**Comment by jackyzha0** [comment]
@SophieMBerger definitely doable :)) I've started implementing the stacked hourglass model in Keras and I'll put up a PR once ive made some progress
